specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2020-12-01/apimtenant.json

```diff
@@ -153,7 +206,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "./definitions.json#/definitions/AccessInformationUpdateParameters"
+              "$ref": "./definitions.json#/definitions/AccessInformationCreateParameters"
```

Comment:
Please use the same definition for the request body and the response. PUT should return what is put in the request (except secrets). You can use `read-only` or `x-ms-mutability` to mark properties in the request or response only.

https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-mutability
https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc008-put-get-patch--list-must-return-the-same-resource-schema